### PR TITLE
Format hidden start time in program-card for Radio Schedules

### LINKED ISSUE #3036: Format hidden start time in program-card for Radio Schedules
**Is your feature request related to a problem? Please describe.**
Hidden start time element that is announcing the start time in the program card is not getting the same timestamp as we have in the visual start time indicator.

**Describe the solution you'd like**
Pass in the correct timestamp to program card component. Hidden start time is a string within `VisuallyHiddenText` so it does not require rendering the whole timestamp element. We just need to transform a unix time into a proper timestamp and pass that string to a program card.

**Describe alternatives you've considered**
N/A

**Testing notes**
Ensure the hidden time stamp and the visual start time indicator both have the same timestamp. 

Dev insight: Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #3036

**Overall change:** 
Formatted the visually hidden start time inside the program card component.
Hidden start time can be seen by checking out developer tools:
Screenshot 2020-02-06 at 12 22 35

For RTL version:
Screenshot 2020-02-10 at 12 28 52

It is also announced by the screen reader correctly:
Screenshot 2020-02-06 at 12 23 45

**Code changes:**
- Used `formatUnixTimestamp` for transforming the timestamp
- Passed `timezone` and `locale` props to program card

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

**Review by tochwill — APPROVED** [review]

File: packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx [hunk 1]
```diff
@@ -127,13 +128,23 @@ const renderHeaderContent = ({
...
 }) => {
   const isOnDemand = state === 'onDemand';
   const isLive = state === 'live';
   const hiddenTextProps = stateLabel === 'Live' ? { lang: 'en-GB' } : {};
 
   const labelWrapperProps = isLive ? { 'aria-hidden': 'true' } : {};
 
+  const formattedStartTime = formatUnixTimestamp(
+    startTime,
+    'HH:mm',
```
Comment (tochwill) [inline packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx:142]: I can't quite remember, we said we're formatting everything as HH:mm for now and then looking at the LT format later on?

Comment (OlgaLyubin) [reply packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx:142]: Yes, that was what we agreed on :)

**Review by DenisHdz — APPROVED** [review]
LGTM 👍

**Review by DenisHdz — COMMENTED** [review]
More conflicts though 🙈

**Comment by PriyaKR** [comment]
Looks good to me the visually hidden timestamp reads out the same time as the visual timestamp.

**Pull request merged**